lang_tests/run.rs

```diff
@@ -3,30 +3,42 @@ use std::{fs::read_to_string, path::PathBuf, process::Command};
 use lang_tester::LangTester;
 use lazy_static::lazy_static;
 use regex::{Regex, RegexBuilder};
-
-// FIXME: doesn't yet test the JIT.
-static BF_BINARIES: &[&str] = &["bf_base"];
+use std::collections::HashMap;
 
 lazy_static! {
     static ref EXPECTED: Regex = RegexBuilder::new("^\\[(.*?)^\\][ \t]*$")
         .multi_line(true)
         .dot_matches_new_line(true)
         .build()
         .unwrap();
+
+    // Maps binary name to a list of benchmarks to skip.
+    static ref BF_BINARIES: HashMap<&'static str, Vec<&'static str>> = HashMap::from([
+        ("bf_base", vec![]),
+        ("bf_simple_yk", vec!["hanoi.bf", "hello.bf", "numwarp.bf", "interp.bf", "bench.bf"]),
+        ("bf_simple2_yk", vec!["hanoi.bf", "hello.bf", "numwarp.bf", "interp.bf", "bench.bf"]),
+    ]);
 }
 
 fn main() {
-    for bf_bin in BF_BINARIES {
+    for (bf_bin, skips) in &*BF_BINARIES {
+        let skips = skips.clone(); // So that the `test_extract` closure can own it.
+        eprintln!("Running tests with interpreter binary: {}", bf_bin);
         LangTester::new()
             .test_dir(".")
-            .test_file_filter(|p| {
-                p.extension().map(|x| x.to_str().unwrap()) == Some("bf")
-            })
-            .test_extract(|p| {
-                EXPECTED
-                    .captures(&read_to_string(p).unwrap())
-                    .map(|x| x.get(1).unwrap().as_str().trim().to_owned())
-                    .unwrap()
+            .test_file_filter(|p| p.extension().map(|x| x.to_str().unwrap()) == Some("bf"))
+            .test_extract(move |p| {
+                if !skips
+                    .clone()
```

Comment:
Is the `clone` necessary?

Reply:
It turns out, it is not. Fixed in fc4b59accae.

Reply:
You've removed a different `clone`! [More clone removal is good, of course, but maybe also look at the one I suggested :)]

Reply:
Yes, we can kill that one too! Fixed in 6040a87.